pom.xml

```diff
@@ -31,7 +31,7 @@
 		<main.basedir>${basedir}</main.basedir>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<java.version>1.8</java.version>
-		<eclipse.repository>https://download.eclipse.org/releases/2019-03/201903201000</eclipse.repository>
+		<eclipse.repository>https://download.eclipse.org/releases/2019-09/</eclipse.repository>
```

Comment:
Thanks for the update. This is the URL of a composite repository which used to result in P2 using http to access the repositories to which the composite refers. Unless that is no longer the case, this URL needs to refer to a "proper" repository. In this case, I think that means the URL should be `https://download.eclipse.org/releases/2019-09/201909181001`

Reply:
Thank you! I have update this URL as you proposed